app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivityViewModel.java

```diff
@@ -39,4 +49,44 @@ public void setFrom(Place from) {
         this.from = from;
     }
 
+    public void setShuttles() {
+        shuttles = appDB.shuttleDao().getAll();
+    }
+
+    public List<Shuttle> getShuttles() {
+        String campusFrom = "";
+        String campusTo = "";
+        if (getFrom() != null && getTo() != null) {
+            if (from.getClass() == Building.class) {
+                campusFrom = ((Building) from).getCampus();
+            } else if (to.getClass() == Building.class) {
+                campusTo = ((Building) to).getCampus();
+            } else if (from.getClass() == Floor.class) {
+                campusFrom = ((Floor) from).getCampus();
+            } else if(to.getClass() == Floor.class) {
+                campusTo = ((Floor) to).getCampus();
+            }
+            if (campusFrom.compareTo(campusTo) == 0) {
+                return null;
+            }
+        }
+
+        Calendar cal = Calendar.getInstance();
```

Comment:
Can you use calendar as variable name instead of cal? it would be clearer for the reader :)